Code/Tools/ProjectManager/Source/ProjectButtonWidget.cpp

```diff
@@ -523,6 +530,15 @@ namespace O3DE::ProjectManager
         ShowMessage(tr("Building Project..."));
     }
 
+    void ProjectButton::ShowExportingState()
+    {
+        m_projectImageLabel->GetShowLogsButton()->show();
+
+        SetProjectExporting(true);
+
+        ShowMessage(tr("Exporting Project..."));
```

Comment:
I think we agreed the terminology is 'Export Launcher' right? Even so, all strings that say 'Export Project' should at least be localized to a common value so there is consistency between what is shown on the menu and what is shown on the log. For example, if you click on 'Export Launcher->Android', the log/status should say 'Exporting Launcher', not 'Exporting Project..'

Reply:
Looking at original script, maybe we should change it back to 'Project', because the cli already is called `export-project`

Reply:
I think we're better off changing it to launcher. In the project manager, there's a lot of terminology around project, so we need the distinction of launcher to clarify what export is doing compared to build